src/foam/u2/PropertyModal.js

```diff
@@ -76,39 +53,33 @@ foam.CLASS({
       name: 'propertyData',
       attribute: true
     },
+    'data'
   ],
 
   methods: [
-    function initE() {
-      this.SUPER();
-      this
-        .addClass(this.myClass())
-        .start()
-          .addClass(this.myClass('main'))
-          .start()
-            .addClass(this.myClass('title'))
-            .add(this.title$)
-          .end()
-          .startContext({ data: this, controllerMode: this.ControllerMode.EDIT })
+    function addBody() {
+      return this.E()
+        .startContext({ controllerMode: this.ControllerMode.EDIT })
           .tag(this.SectionedDetailPropertyView, {
             prop: this.property,
             data$: this.data$
           })
-          .endContext()
-        .end()
-        .start()
-          .addClass('buttons')
-          .startContext({ data: this })
-            .tag(this.CANCEL, { buttonStyle: 'SECONDARY' })
-            .tag(this.OK)
-          .endContext()
-        .end();
+        .endContext();
+    },
+    function addActions() {
+      var actions = this.E().startContext({ data: this });
+      actions.tag(this.OK, { isDestructive: this.modalStyle == 'DESTRUCTIVE' });
```

Comment:
So default actions are OK and  CANCEL?

Reply:
Yeah and they can be overridden just like they were before